.github/workflows/ci.yml

```diff
@@ -56,6 +56,13 @@ jobs:
         components: rustfmt
     - run: cargo fmt --all -- --check
 
+  runefmt:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: dtolnay/rust-toolchain@stable
+    - run: shopt -s globstar ; cargo run --bin rune -- fmt --check **/*.rn
```

Comment:
I'd like to encapsulate all the examples in the scripts folder in the workspace somehow, that way this could be called without globbing. But this obviously works for now!

Reply:
I think it'd be nice with a `-R` flag or something on the script; just felt like it'd need some more changes to the CLI entrypoint.

Reply:
Just remembered that recursive does exist with `--recursive`.